Cassette/CASFileObjectQueue.m

```diff
@@ -47,7 +47,21 @@ - (NSUInteger)size {
 }
 
 - (void)add:(id)data {
-    [self.queueFile add:[NSKeyedArchiver archivedDataWithRootObject:data]];
+    NSError *error;
+    NSData *serializedData;
+    if (@available(iOS 11.0, macOS 10.13, *)) {
```

Comment:
if we are supporting watchOS & tvOS, I think we need to also specify
`if (@available(iOS 11.0, macOS 10.13, watchOS 4.0, tvOS 11.0, *)`
We may need to add this for the unarchive as well

Reply:
actually on further discussion it turns out we do not supports watchOS and tvOS currently, so resolving.